content-src/components/MediaPreview/MediaPreview.js

```diff
@@ -0,0 +1,43 @@
+const React = require("react");
+const classNames = require("classnames");
+
+const MediaPreview = React.createClass({
+  getInitialState() {
+    return {
+      showPlayer: false
+    };
+  },
+
+  getDefaultProps() {
+    return {
+      site: {},
```

Comment:
You should define `PropTypes` for `MediaPreview` so that you incorrect prop definitions will throw errors in tests/create warnings. It would probably be a good idea to use [`React.PropTypes.shape`](https://facebook.github.io/react/docs/reusable-components.html) to define the shape of the `site` prop.